surface/include/pcl/surface/ball_pivoting.h

```diff
@@ -214,8 +305,10 @@ namespace pcl
      * @param num_point_in_radius
      * @param ratio_success
      */
-    virtual void
-    setEstimatedRadius (const int num_sample_point, const int num_point_in_radius, const float ratio_success);
+    void
+    setSearchRadiusAutomatically (const int num_sample_point = 0, 
```

Comment:
This definitely needs to be documented. What if instead of number of sample points this function take the fraction of sample points, between 0 and 1? And the default value will be explicitly 0.2?